test/test_grudge_sym_old.py

```diff
@@ -148,7 +148,7 @@ def _get_variables_on(dd):
 
     num_integral_2 = np.dot(f_volm, actx.to_numpy(flatten(mass_op(f=ones_quad))))
     err_2 = abs(num_integral_2 - true_integral)
-    assert err_2 < 2e-9, err_2
+    assert err_2 < 4e-9, err_2
```

Comment:
How did this fix
```
assert 5.609763320535421e-09 < 2e-09
```
? I'm increasingly confused :cry:

Reply:
Argh. That's the wrong one, isn't it? Dang. I'll do another PR.

I think the truth is that Github CI serves up a happy hodgepodge of different machines, all with interesting and different FP behavior.

Reply:
#248

Reply:
Haha, yeah, that sounds quite likely. Thanks!